src/Mailgun/Api/MailingList.php

```diff
@@ -0,0 +1,268 @@
+<?php
+
+/*
+ * Copyright (C) 2013 Mailgun
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+namespace Mailgun\Api;
+
+use Mailgun\Assert;
+use Mailgun\Model\MailingList\PagesResponse;
+use Mailgun\Model\MailingList\CreateResponse;
+use Mailgun\Model\MailingList\ShowResponse;
+use Mailgun\Model\MailingList\UpdateResponse;
+use Mailgun\Model\MailingList\DeleteResponse;
+use Mailgun\Model\MailingList\MembersResponse;
+use Mailgun\Model\MailingList\CreateMemberResponse;
+use Mailgun\Model\MailingList\ShowMemberResponse;
+use Mailgun\Model\MailingList\UpdateMemberResponse;
+use Mailgun\Model\MailingList\DeleteMemberResponse;
+use Psr\Http\Message\ResponseInterface;
+
+/**
+ * @author Michael Münch <helmchen@sounds-like.me>
+ */
+class MailingList extends HttpApi
+{
+    /**
+     * Returns a paginated list of mailing lists on the domain.
+     *
+     * @param int $limit Maximum number of records to return (optional: 100 by default)
+     *
+     * @return PagesResponse|array|ResponseInterface
+     *
+     * @throws \Exception
+     */
+    public function pages($limit = 100)
+    {
+        Assert::integer($limit);
+
+        $params = compact('limit');
+
+        $response = $this->httpGet('/v3/lists/pages', $params);
+
+        return $this->hydrateResponse($response, PagesResponse::class);
+    }
+
+    /**
+     * Creates a new mailing list on the current domain.
+     *
+     * @param string $address      Address for the new mailing list
+     * @param string $name         Name for the new mailing list (optional)
+     * @param string $description  Description for the new mailing list (optional)
+     * @param string $access_level List access level, one of: readonly (default), members, everyone
+     *
+     * @return CreateResponse|array|ResponseInterface
+     *
+     * @throws \Exception
+     */
+    public function create($address, $name = null, $description = null, $access_level = 'readonly')
```

Comment:
PHP variables should not be defined as `snake_case` (`$access_level`), they should be `camelCase` (`$accessLevel`).